### ISSUE TITLE
when train_backbone，got a bad performance!

### ISSUE THREAD

**Issue by PHaiJun** [DonaldRR/SimpleNet#47]
why??

INFO:__main__:instance_auroc: 0.682
INFO:__main__:full_pixel_auroc: 0.661
INFO:__main__:anomaly_pixel_auroc: 0.007
INFO:__main__:

-----

INFO:utils:instance_auroc: 0.682
INFO:utils:full_pixel_auroc: 0.661
INFO:utils:anomaly_pixel_auroc: 0.007

**Comment by PHaiJun** [comment]
@DonaldRR

**Comment by DonaldRR** [comment]
The FREEZING backbone is one of the key in this work. The pretrained backbone models the "normal" features distribution. When it's parameters are changing over time, such distribution will be changing too.

**Comment by DonaldRR** [comment]
The essence of the training is not always Decreasing the Loss. The freezing backbone makes the features from backbone are meaningful,

**Comment by sdracula** [comment]
> @DonaldRR

excuse me, how could you get the good performence by this resposiry
